addons/l10n_dk/data/account_tax_template_data.xml

```diff
@@ -101,7 +101,7 @@
     <record id="tax310" model="account.tax.template">
         <field name="chart_template_id" ref="dk_chart_template"/>
         <field name="sequence">310</field>
-        <field name="name">3. Land Salg Vare / Ydelser</field>
+        <field name="name">3. Country Sales Goods/Services</field>
```

Comment:
You used 3rd country earlier for the fiscal position, would be nice if it was the same for both